test/sharness/t0141-addfilter.sh

```diff
@@ -27,7 +27,7 @@ test_swarm_filter_cmd() {
 	'
 
 	test_expect_success "'ipfs swarm filters' output looks good" '
-		test_sort_cmp list_actual list_expected
```

Comment:
This can be all killed in 1 commit.
From `grep`:

```
t0141-addfilter.sh
30:		test_sort_cmp list_actual list_expected

t0120-bootstrap.sh
37:    test_cmp list_actual list_expected
45:    test_cmp list2_actual list_expected
66:    test_cmp add_actual add_expected
78:    test_cmp rm_actual rm_expected
97:    test_cmp add2_actual add2_expected
116:    test_cmp rm2_actual rm2_expected

t0081-repo-pinning.sh
104:	test_cmp actual1 expected1
234:	test_cmp actual8 expected8

xt0130-multinode.sh
33:	test_cmp actual1 expected1

x0045-add-cat-iptb.sh
33:	test_cmp actual1 expected1
```